dev/set_matrix.py

```diff
@@ -168,18 +184,23 @@ def uploaded_recently(dist: dict[str, Any]) -> bool:
 def get_released_versions(package_name: str) -> list[Version]:
     data = pypi_json(package_name)
     versions: list[Version] = []
-    for version, distributions in data["releases"].items():
+    for version_str, distributions in data["releases"].items():
         if len(distributions) == 0 or any(d.get("yanked", False) for d in distributions):
             continue
 
-        # Ignore versions that were uploaded recently to avoid testing unstable
-        # versions. Newly released versions often contain undiscovered bugs
-        # (example: https://github.com/huggingface/transformers/issues/34370).
-        if any(map(uploaded_recently, distributions)):
```

Comment:
In the old code, `max: 1.21.0` is not allowed because we exclude versions released within a day.